gulpfile.js

```diff
@@ -101,15 +100,14 @@ gulp.task('webpack:watch:scss-to-json', (cb) => {
 /**
  * Webpack config and setup.
  */
-// URL to visit to see local PL
-const localhost = 'http://localhost:8080';
 // Import webpack config for PL
 const wpconfig = require('./webpack.pl.config');
 // Webpack Dev Server config used for local development.
 // See all available config options:
 // https://webpack.js.org/configuration/dev-server/#devserver
 const serverconfig = {
-  publicPath: url.resolve(localhost, wpconfig.output.publicPath), // ie http://localhost:8080/temp
+  host: '0.0.0.0',
+  port: '8080',
```

Comment:
This can be overridden from webpack.pl.config?

Reply:
There is zero webpack dev server config in our webpack configs. It's either handled at our [webpack-dev-server.js](https://github.com/phase2/pattern-lab-starter/pull/133/files#diff-1763473eec6d663f46bbaecfdf53009e) service or within this gulpfile.js task.

Reply:
Well, the thing is that binding to 0.0.0.0:80 is really aggressive outside the context of a container.